deployer/scripts/preflight.sh

```diff
@@ -27,6 +27,20 @@ function validate_master_accessible() {
   return 1
 }
 
+function validate_hostname() {
+  #The route will only accept RFC 952 based hostnames
+  if [[ $hawkular_metrics_hostname =~ ^[a-zA-Z][a-zA-Z0-9.-]?+[a-zA-Z0-9]$ ]]; then
+    echo "ok"
```

Comment:
since we are echoing here maybe we want to echo something more then ok...like "validated hostname is an acceptible...."